src/nullshield.ts

```diff
@@ -1,2 +1,2 @@
-export { default as OptionT } from './OptionT';
-export { default as ResultT } from './ResultT';
+export { default as Optional } from './Optional';
+export { default as Result } from './Result';
```

Comment:
Would it be worth _also_ exporting OptionT, and ResultT for a while and saying theyre deprecated? 

Maybe theres a way that we could export them here but wrap them somehow so that they issue warnings on the console about the deprecation and allow you to update?

Reply:
Well, we _could_, but I feel like it's not that necessary.  Maybe this is too strong of a stance to take, but since we're still pre-1.0 I really feel like we shouldn't worry about this type of thing too much.

Would a good compromise be to put out a release notice with a migration guide?  The steps to upgrade here are going to be pretty minimal.  Should just be a find+replace for both of the types listed above.

Reply:
@ryanguill cc-ing you here, not sure if review comment even send out notifications or not.

Reply:
yeah, probably not necessary.